python/ray/tests/test_get_locations.py

```diff
@@ -0,0 +1,78 @@
+import numpy as np
+import pytest
+import platform
+import time
+
+import ray
+
```

Comment:
Still trying to figure out the best practice for testing. any comments/suggestions are appreciated!